.env.example

```diff
@@ -13,3 +13,5 @@ SYSTEM_ROLE="Your role is to assist the user with work-related tasks, such as se
 EMAIL_WHITELIST=kate@hotmail.com,bob@hotmail.com,@scottlogic.com
 # Session secret 
 SESSION_SECRET=YOUR_SESSION_SECRET
+# Path to document folder
+DOCUMENT_FOLDER=backend/resources/documents
```

Comment:
I think because this is unlikely to need to be configured, it would be better to have this as a `const` in the code rather than an environment variable.